src/views/Peers.tsx

```diff
@@ -71,22 +77,28 @@ export const Peers = () => {
     const actionsMenu = (<Menu items={itemsMenuAction} ></Menu>)
 
     const transformDataTable = (d:Peer[]):PeerDataTable[] => {
-        return d.map(p => ({ key: p.IP, ...p } as PeerDataTable))
+        const peer_ids = d.map(_p => _p.ID)
+        return d.map((p) => {
+            const gs = groups
+                .filter(g => g.Peers?.find((_p:GroupPeer) => _p.ID === p.ID))
+                .map(g => ({ID: g.ID, Name: g.Name, PeersCount: g.Peers?.length, Peers: g.Peers || []}))
+            return {
```

Comment:
Codacy has a fix for the issue: [Insert `}));⏎··`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
        }));
```